frontend/ui/widget/frontlightwidget.lua

```diff
@@ -298,23 +293,12 @@ function FrontLightWidget:addWarmthWidgets(num_warmth, step, vertical_group)
         end
     end
 
-    if self.powerd.auto_warmth then
-        enable_button_plus = false
-        enable_button_minus = false
-        button_color = Blitbuffer.COLOR_DARK_GRAY
-    else
-        if math.floor(num_warmth / self.nl_scale) <= self.nl_min then enable_button_minus = false end
-        if math.ceil(num_warmth / self.nl_scale) >= self.nl_max then enable_button_plus = false end
```

Comment:
IIRC, this ought to be kept (instead of using `self.fl_max` as you did below).

Reply:
Yes, that is better. I shift the two lines down, so that they are nearer to where the enable_button_xxx are used.